python/cugraph-pyg/cugraph_pyg/tests/conftest.py

```diff
@@ -79,7 +78,7 @@ def karate_gnn():
     el = karate.get_edgelist().reset_index(drop=True)
     el.src = el.src.astype("int64")
     el.dst = el.dst.astype("int64")
-    all_vertices = np.array_split(cudf.concat([el.src, el.dst]).unique().values_host, 2)
+    all_vertices = np.array_split(np.arange(34), 2)
```

Comment:
This failure is probably caused by https://github.com/rapidsai/cudf/pull/11656.

The ordering of cudf's `unique` and `drop_duplicates` has changed. Previously, the order was _sorted_. Now, the order is _stable_ (preserves input order while removing duplicates). This may affect other parts of cuGraph, too, so I will take a look.

Reply:
Right, I was aware of that.  Ironically, I have another PR which takes advantage of the new default behavior for `drop_duplicates`.

Reply:
Yes, it looks like `unique` is heavily used in cuGraph's Python code. We may need to review these for correctness if sorted order is expected. https://github.com/search?q=repo%3Arapidsai%2Fcugraph+unique%28+language%3APython&type=code&l=Python

Reply:
Awesome, thanks @alexbarghi-nv! Glad to hear the new behavior is helpful for your other PR! :D